StripeApplePay/StripeApplePay/Source/ApplePayContext/STPApplePayContext.swift

```diff
@@ -107,7 +107,11 @@ public class STPApplePayContext: NSObject, PKPaymentAuthorizationControllerDeleg
         delegate: _stpinternal_STPApplePayContextDelegateBase?
     ) {
         STPAnalyticsClient.sharedClient.addClass(toProductUsageIfNecessary: STPApplePayContext.self)
-        if !StripeAPI.canSubmitPaymentRequest(paymentRequest) {
+        if paymentRequest.merchantIdentifier.isEmpty {
+            return nil
+        }
+        // "In versions of iOS prior to version 12.0 and watchOS prior to version 5.0, the amount of the grand total must be greater than zero."
+        if !(paymentRequest.paymentSummaryItems.last?.amount.floatValue ?? 0.0 >= 0) {
```

Comment:
We only support iOS 13 and above, so I don't think we need this check

Reply:
Hi @sfriedman-stripe. Thank you for the feedback.
The comment you indicated comes from the method which was previously invoked, which you can find here: https://github.com/stripe/stripe-ios/blob/0355a45caaf0ce9dd03c6144efda17ea38d6a0be/StripeCore/StripeCore/Source/API%20Bindings/StripeAPI.swift#L86

I can remove it if you'd like, no problem!